app/models/miq_cockpit_ws_worker/runner.rb

```diff
@@ -151,7 +151,7 @@ def cockpit_ws_run
     Bundler.with_clean_env do
       stdin, stdout, stderr, wait_thr = Open3.popen3(env, *cockpit_ws.command(BINDING_ADDRESS), :unsetenv_others => true)
     end
-    stdin.close
+    stdin&.close
```

Comment:
I believe the intention was to have `stdin.close` in the scope of the `Open3` command and was accidentally moved to a different scope, outside the `Bundler.with_clean_env` scope,  in commit: https://github.com/ManageIQ/manageiq/commit/63b44ed6d0a69eec3d9d8a31cd8f455cbdce1917

I believe checking for `stdin` here is not right.  We should either not close `stdin` at all or move the `stdin.close` in the `with_clean_env` block.

cc @carbonin

Reply:
> move the stdin.close in the with_clean_env block.

:point_up: This I think.